### ISSUE TITLE
[BOM] Shared ressources checkbox is not working anymore

### ISSUE THREAD

**Issue by RachL** [openfoodfoundation/openfoodnetwork#12099]
## Description

According to the user guide, we should be able to edit quantities of different variants of the same product thanks to this checkbox.

https://guide.openfoodnetwork.org/basic-features/orders/view-orders#the-shared-resource-checkbox

Now it's not showing up any differences.

## Steps to Reproduce

1. Log in as enterprise admin
2. Got to `/admin/orders/bulk_management`
3. Select a product
4. Play with the checkbox

## Workaround
Edit individually

## Severity
bug-s3: a feature is broken but there is a workaround

## Your Environment

* Version used: v4.4.27
* Browser name and version: Firefox and Chrome
* Operating System and version (desktop or mobile): both

**Comment by cyrillefr** [comment]
Hello @RachL ,

I have tried to figure out this issue.
There is no JavaScript issue (no problem in console).
From a newbie user (like me), I don't understand what should I do to the bulk edit.
There are no hidden fields or anything that appear on clicking the checkbox;
The doc seems a bit light on that.

So I searched when the code was implemented at first.
I have found it here (dates back from 2014):
https://github.com/openfoodfoundation/openfoodnetwork/commit/cd1086ce051a2abffd5a8b201dea48b1e4e3131c#diff-70474db1fb8f4bf7ce9188f6ae70d72bdb31f5a1eb175cc6666e9a0e3d2af29b

I have looked at the tests and found no descriptive test that could enlighten me about that bulk editing.
Hope I am not missing anything here.

Also I haven't found the corresponding Issue/PR to get some more insights.

Has this feature ever worked ?

**Comment by RachL** [comment]
@cyrillefr thanks for that search!

Yes the person who did the screenshot for the user guide confirmed to me that it was working. If you see the screenshot, I believe `tomatoes: 3kg` and `tomatoes: 1kg` are showing up because of the checkbox

![bomtomshared](https://github.com/openfoodfoundation/openfoodnetwork/assets/6525576/992eef4a-0c06-4d04-83aa-63a3350f3b12)

otherwise, to end up with a view filtered on a product you need to use the search or click on the name. But you would only have one of the variant, not both. Does it make sense?

@tschumilas @kirstenalarsen maybe you have more insights on how it was supposed to work?

**Comment by tschumilas** [comment]
yes this issue is still here.  My understanding is that checking the 'shared resource' box SHOULD filter the selection differently.  When it is checked -  different variants (usually different sizes as in Rache's example) should show.  When the box is not checked,  only the specific filtered variant should show.  Right now checking /unchecking the shared resource box does not change the filtered display.
